src/test/scala/ai/eto/rikai/sql/model/mlflow/SparkSessionWithMlflow.scala

```diff
@@ -57,7 +63,18 @@ trait SparkSessionWithMlflow extends LazyLogging with BeforeAndAfterAll {
 
   lazy val mlflowClient = new MlflowClientExt(testMlflowTrackingUri)
 
-  private[mlflow] def clearModels(): Unit = {
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    run = mlflowClient.client.createRun()
+  }
+
+  override def afterEach(): Unit = {
+    mlflowClient.client.deleteRun(run.getRunId)
+    clearModels()
```

Comment:
Ok is the purpose of this test class so we can do the required setup/teardown?

Reply:
It clears mlflow as well as the Hive Catalog to a clean state.